Update for KSP 1.8+

Tutul- opened the pull request:
Work fine on KSP 1.7.3, without any visible error or crash and produce the desired effects. So I propose to update it to the latest KSP version to enable CKAN user to use it without the need for "KSP version compatibility override".

I also update LineRender initialization to remove the deprecated function calls.

Tutul- committed:
Update for KSP 1.7.9

Work fine on KSP 1.7.3, without any visible error or crash and produce the desired effects. So I propose to update it to the latest KSP version to enable CKAN user to use it without the need for "KSP version compatibility override".

I also update LineRender initialization to remove the deprecated function calls.

Tutul- commented:
Work fine with other mods like EVE/Scatterer

MJavad commented:
I get crashes after loading screen with this mod, not sure if it's compatible with 1.7.3.

Tutul- commented:
I don't have any problems with it and I use a lot of mods with it, here is my list:

- KerbalAlarmClock
- KerbalJointReinforcementContinued
- kOS
- Trajectories
- ModuleManager
- Spectra
- DistantObject
- RealPlume
- PlanetShine
- KabramsSunFlaresPack-Orange-High
- DistantObject-default
- EnvironmentalVisualEnhancements
- Scatterer
- Scatterer-config
- TextureReplacer
- Kopernicus
- ModularFlightIntegrator
- RealPlume-StockConfigs
- SmokeScreen
- Kerbalism
- Kerbalism-Config-Default
- CommunityResourcePack
- HeatControl
- RemoteTech
- CommunityTechTree
- Chatterer
- BetterBurnTime
- DMagicOrbitalScience
- UniversalStorage2
- SCANsat
- NearFutureConstruction
- NearFuturePropulsion
- NearFutureSolar-Core
- StationPartsExpansionRedux
- B9PartSwitch
- NearFutureSolar
- NearFutureProps
- RemoteTechStockConfigs
- HideEmptyTechNodes
- MissingHistory
- WaypointManager
- ContractConfigurator-CleverSats
- ContractConfigurator-KerbalAcademy
- ContractConfigurator-KerbinSpaceStation
- ContractConfigurator-RemoteTech
- ContractConfigurator
- UnKerballedStart
- CustomBarnKit
- QuickCursorHider
- KSP-AVC
- ConnectedLivingSpace

Tutul- commented:
Can you check your logs to see if the crash is comming from him, from him with an other mods or maybe a special case with him alone ?

Tutul- committed:
Update to use Framework 4.0 for KSP 1.8+

Since some planet texture and ambiance has changed, we may need to update color configuration. I didn't test it on other planet yet!

Tutul- commented:
Seem to load but need further testing.
As some planet get their textures/color/atmosphere changed, we need to check if planetside's color need to be updated.

May still cause instability as I didn't test that big upgrade on a long run. I want to test his full stability with some other mods to detect any problems but two aren't yet updated.

I'll keep you inform
